app/scripts/migrations/091.ts

```diff
@@ -0,0 +1,55 @@
+import { hasProperty, isObject } from '@metamask/utils';
+import { cloneDeep } from 'lodash';
+
+export const version = 89;
+
+/**
+ * Add an `id` to the `providerConfig` object.
```

Comment:
Is this an accurate description of the migration?

Reply:
fixed in 4f53b4cb10